examples/bri-3/src/bri/zeroKnowledgeProof/models/proof.ts

```diff
@@ -0,0 +1,5 @@
+export class Proof {
+  a: string[];
```

Comment:
not sure what does a,b,c signify. Either change these variable names, or add a comment about what they signify.

Reply:
Fixed.